src/vos/vos_io.c

```diff
@@ -1659,6 +1661,70 @@ akey_update_recx(daos_handle_t toh, uint32_t pm_ver, daos_recx_t *recx,
 	return rc;
 }
 
+int
+vos_key_mark_agg(struct umem_instance *umm, struct vos_krec_df *krec)
```

Comment:
Just curious, why the flags for key are not stored in the btr or evt root feats like object/container? It looks to me not much difference but may save few bits for the 'kr_bmap'.

Reply:
I suppose it could be done this way.  Let me look into it.   The main reason was for simplicity but yeah, kr_bmap is limited